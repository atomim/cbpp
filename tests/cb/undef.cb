#define MAGIC
#ifdef MAGIC
	AddText "magic!"
#endif

#undef MAGIC

#ifdef MAGIC
	AddText "maagic"
#endif

AddText "this should be normal"