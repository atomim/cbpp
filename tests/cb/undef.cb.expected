// #define MAGIC
// #ifdef MAGIC
	AddText "magic!"
// #endif

// #undef MAGIC





AddText "this should be normal"